GitCommands/AppTitleGenerator.cs

```diff
@@ -21,18 +22,19 @@ public interface IAppTitleGenerator
     /// <summary>
     /// Generates application title.
     /// </summary>
+    [Export(typeof(IAppTitleGenerator))]
     public sealed class AppTitleGenerator : IAppTitleGenerator
     {
 #if DEBUG
         private static string? _extraInfo;
 #endif
 
-        private readonly IRepositoryDescriptionProvider _description;
-
-        public AppTitleGenerator(IRepositoryDescriptionProvider description)
-        {
-            _description = description;
-        }
+#pragma warning disable CS0649
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
+        [Import(typeof(IRepositoryDescriptionProvider))]
+        private IRepositoryDescriptionProvider _description;
```

Comment:
💡 I highly recommend using constructor injection instead of property/field injection.